src/pages/deadlines/DeadlinesPage.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { Box } from "native-base";
+import { DrawerScreenProps } from "@react-navigation/drawer";
+import { RootDrawerParamList } from "../../interfaces/navigatorInterfaces";
+import { CalendarList } from "react-native-calendars";
+import { useAppSelector } from "../../redux";
+import { isAssignment } from "../../utils/course";
+
+type Props = DrawerScreenProps<RootDrawerParamList, "Deadlines">;
+
+const DeadlinePage = ({ navigation }: Props) => {
+  const modules = useAppSelector((state) => state.modules);
+  const courses = useAppSelector((state) => state.courses);
+  const deadlines = modules.filter((mod) => isAssignment(mod.type));
```

Comment:
Please add some more filter conditions that:

1) make sure that the due date is non-null, if it is null, then assign it as today
2) make sure that the deadline has not been marked removedFromMoodle or userMarkedDeleted

Reply:
If null due dates are assigned to today then the calendar would display dots for all of them on today's date. Better to ignore null values right?

Reply:
@pranay1208 Take a look now